src/emm/emm_cli.py

```diff
@@ -98,6 +98,19 @@ def git_commit_modules(self, commit_message: str) -> None:
         """
         self.modules_service.git_commit_modules(commit_message)
 
+    def git_pull_request(self, args: List[str]) -> None:
+        """
+        Create pull request to all modules.
+
+        :param args: Arguments pass to the github CLI.
+        """
+        authenticated = self.modules_service.validate_github_authentication()
+        if not authenticated:
+            print("Please authenticate github CLI.")
```

Comment:
I run the github CLI on the front process, it would display 
```
You are not logged into any GitHub hosts. Run gh auth login to authenticate.
```
before the exception, I think that's enough information for the user? 
We can just add a clickException as a reminder and early termination.

Reply:
If that is the case, can you add a comment in the code mentioning it? And we should still make sure the command exits properly (with a non-zero code).